project-base/scripts/install.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+numberRegex='^[0-9]+([.][0-9]+)?$'
+operatingSystem=''
+allowedValues=(1 2)
+declare -A operatingSystemsByIndex
+
+operatingSystemsByIndex[1]='Linux'
+operatingSystemsByIndex[2]='Mac'
+
+echo This is installation script that will install demo Shopsys Framework application on docker with all required containers and with demo database created.
+
+echo "Start with specifying your operating system: \
+    1) Linux
+    2) Mac
+    "
+
+while [[ 1 -eq 1 ]]
+do
+    read -p "Enter OS number: " operatingSystem
+    if [[ ${operatingSystem} =~ $numberRegex ]] ; then
+        if [[ " ${allowedValues[@]} " =~ " ${operatingSystem} " ]]; then
+            break;
+        fi
+        echo "Not existing value, please enter one of existing values"
+    else
+        echo "Please enter a number"
+    fi
+done
+
+echo "You chosed ${operatingSystemsByIndex[$operatingSystem]}. Now your app will be installed.."
+
+echo "Creating config files.."
+cp -f app/config/parameters.yml.dist app/config/parameters.yml
```

Comment:
is this needed? can you check whether during composer install is that copied automatically from `.dist` files ?